Deployment/Milyli.ScriptRunner.Services.Interfaces.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Milyli.ScriptRunner.Services.Interfaces</id>
+	<version>1.0.0.0</version>
+    <title>ScriptRunner Services</title>
+    <authors>Milyli inc</authors>
+    <projectUrl>https://github.com/Milyli/RelativityScriptRunner</projectUrl>
+    <owners>Milyli inc</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Kepler services to interact with Milyli ScriptRunner</description>
+  </metadata>
+  <files>
+    <file src=".\Artifacts\Milyli.ScriptRunner.Services.Interfaces.dll"
+	target="lib\Milyli.ScriptRunner.Services.Interfaces\net462" />
+    <file src=".\Artifacts\Milyli.ScriptRunner.Services.Contracts.dll"
+	target="lib\Milyli.ScriptRunner.Services.Interfaces\net462" />
+    <file src=".\Artifacts\Milyli.ScriptRunner.Services.Contracts.dll"
```

Comment:
- [ ] Services.Contracs.dll is included twice
  - [ ] The README isn't a lib file and should go in the docs folder
  - [ ] The Milyli.ScriptRunner.Services.Interfaces folder probably isn't needed and the dlls can just go in lib\net462